components/email/src/Email.svelte

```diff
@@ -353,15 +353,9 @@
       //#endregion read/unread
 
       const lastMsgIndex = activeThread.messages.length - 1;
-      activeThread.messages[lastMsgIndex].expanded =
-        !activeThread.messages[lastMsgIndex].expanded;
-
-      if (!emailManuallyPassed) {
-        // fetch last message
-        if (!activeThread.messages[lastMsgIndex].body) {
-          fetchIndividualMessage(lastMsgIndex);
-        }
-      }
```

Comment:
removed `fetchIndividualMessage` calls in favour of having it inline with HTML. 

This is because there isn't a great way to tell Email component to get the last message when `show_expanded_email_view_onload` is on.